test/Templates.Test/EmptyWebTemplateTest.cs

```diff
@@ -14,7 +14,8 @@ public EmptyWebTemplateTest(ITestOutputHelper output) : base(output)
         }
 
         [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Is OR'ing these not valid at all? Should we change this attribute to fail if it's used incorrectly?

Reply:
It does not work. Not sure why it was being used. Filed an issue [here](https://github.com/aspnet/Home/issues/2803)